tlg-base-config_3-vm/azuredeploy.json

```diff
@@ -0,0 +1,536 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "configName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the deployment, i.e. BaseConfig01. Do not use spaces or special characters other than _ or -. Used to concatenate resource names for the deployment."
+      },
+      "defaultValue": "TlgBaseConfig-01"
+    },
+    "domainName": {
+      "type": "string",
+      "metadata": {
+        "description": "The FQDN of the new AD domain."
+      },
+      "defaultValue": "corp.contoso.com"
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the domain administrator account to create, i.e. globaladmin. This will also be used to create a local admin account on each VMs."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The administrator account password."
+      }
+    },
+    "deployClientVm": {
+      "type": "string",
+      "allowedValues": [
+        "No", "Yes"
+      ],
+      "metadata": {
+        "description": "Choose 'No' to deploy no client VM, or 'Yes' if you want to deploy a client VM from a custom generalized VHD that you have uploaded to an accessible storage account."
+      },
+      "defaultValue": "No"
+    },
+    "clientVhdStorageAccount": {
```

Comment:
I eliminated the vhdName param, but the SAS key needs to be passed to a child template as a securestring.

Reply:
Right - but you can combine the base uri and the sastoken to make it easier to deploy - rather than having the user split them and the template concat them again

Reply:
Just to be clear, you want me to ask for the entire URI including sastoken, and create a variable that splits it at the "?" char?